demos/projects/ST/b-l475e-iot01a/config/demo_config.h

```diff
@@ -505,6 +505,18 @@ static unsigned char root_cert_array[] = {
  *  WIFI_ECN_WPA2_PSK = 0x03,
  *  WIFI_ECN_WPA_WPA2_PSK = 0x04
  */
-#define WIFI_SECURITY_TYPE               WIFI_ECN_WPA2_PSK
+#define WIFI_SECURITY_TYPE                   WIFI_ECN_WPA2_PSK
+
+#ifdef democonfigENABLE_DPS_SAMPLE
+    #define democonfigCHUNK_DOWNLOAD_SIZE    256
+#else
+    #define democonfigCHUNK_DOWNLOAD_SIZE    1024
+#endif /* democonfigENABLE_DPS_SAMPLE */
+
+#define democonfigADU_DEVICE_MANUFACTURER    "STMicroelectronics"
+#define democonfigADU_DEVICE_MODEL           "STM32L475"
+#define democonfigADU_UPDATE_PROVIDER        "STMicroelectronics"
```

Comment:
Should these include some information about our SDK and version for telemetry purposes?

Reply:
This is what shows up in Azure Portal when you select device information, so I think it's probably better to be just about the device. We could definitely open this as a topic for all of the ADU samples (currently the rest of them are similar to this)

Reply:
In that case it should definitely be something like `Azure IoT SDK FreeRTOS ADU Sample` (or similar w/o spaces) to make sure there is no confusion between prod and test devices.

Reply:
I'll open a work item for this and the printing issue to solve these issues in a consistent way across the samples